src/chain/interfaces.ts

```diff
@@ -20,29 +20,31 @@ import type statemint from './statemint/api'
 import type subsocial from './subsocial/api'
 
 
+// Enable only Kusama network to be type safe during development
 export type ChainApi =
-  | typeof acala
-  | typeof astar
-  | typeof bifrost
-  | typeof calamari
-  | typeof efinity
-  | typeof gmordie
-  | typeof hydradx
-  | typeof karura
-  | typeof khala
+  // | typeof acala
+  // | typeof astar
+  // | typeof bifrost
+  // | typeof calamari
+  // | typeof efinity
+  // | typeof gmordie
+  // | typeof hydradx
+  // | typeof karura
+  // | typeof khala
   | typeof kusama
-  | typeof moonbeam
-  | typeof moonriver
-  | typeof phala
-  | typeof polkadot
-  | typeof shiden
-  | typeof statemine
-  | typeof statemint
-  | typeof subsocial
+  // | typeof moonbeam
+  // | typeof moonriver
+  // | typeof phala
+  // | typeof polkadot
+  // | typeof shiden
+  // | typeof statemine
+  // | typeof statemint
+  // | typeof subsocial
```

Comment:
forgotten or?

Reply:
Since, we have only Kusama network working right now, there are no types generated for these networks and typescript is confused when you work the generic `chain` variable

Reply:
Can you leave like a TODO or smth similar at the top so that it's not just a file with 95% commented out code.

Reply:
there is already a comment `// Enable only Kusama network to be type safe during development`. If its not descriptive enough, i can add another one

Reply:
Yea, you could add a short description about why we currently don't want to make the other networks type safe and that it is planned for the future. Might want to start the comment with "TODO:" because some editors have build in functionalities looking for todos.

Reply:
done